src/main/java/edu/kis/powp/command/ShapeFactory.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ShapeFactory{
+	public static void drawSquare(Job2dDriver driver){
+		ComplexCommand complex = new ComplexCommand();
+		complex.addCommand(new SetPositionCommand(20, 20, driver));
+		complex.addCommand(new OperateToCommand(30, 20, driver));
+		complex.addCommand(new OperateToCommand(30, 30, driver));
+		complex.addCommand(new OperateToCommand(20, 30, driver));
+		complex.addCommand(new OperateToCommand(20, 20, driver));
+		complex.execute();
```

Comment:
This is not a factory. A factory should return a driver command (not execute it). Missing point 3.1.5.